regress/sql/expr.sql

```diff
@@ -2107,6 +2107,69 @@ SELECT * from cypher('expr', $$
     RETURN sqrt("1")
 $$) as (result agtype);
 
+
+--
+-- fact()
+--
+-- Factorial function tests age_fact()
+-- 0! = 1
+SELECT * FROM age_fact(0);
+-- 1! = 1
+SELECT * FROM age_fact(1);
+-- 5! = 120
+SELECT * FROM age_fact(5);
+-- 10! = 3628800
+SELECT * FROM age_fact(100);
+-- 20! = 2432902008176640000
+SELECT * FROM age_fact(500);
+
+
+-- Overflow test
+SELECT * FROM age_fact(32178);
+
+
+-- Inside the cypher query
+-- At CREATE clause
+SELECT * FROM cypher('expr', $$
```

Comment:
The regression tests for CREATE and MATCH are run after EXPR, and generally you should let those specific tests run before using them, the above tests are satisfactory

Reply:
Kept the above and removed the rest.